examples/leasing/node_release_script.py

```diff
@@ -0,0 +1,205 @@
+#! /bin/python
+"""
+This script will free nodes from a project after
+a specific amount of time.
+Script reads a config file from /etc/leasing.cfg
+which defines which nodes are in the project, time threshold
+admin user name and password and status file.
+
+status file includes each node's status, which node is in
+the project, which one is free and for how long the node
+has been in the project.
+
+*******************
+
+Config file format (/etc/leasing.cfg):
+node_list: names of the nodes
+threshold: number of times that cron job is called before
+            releasing the node
+user_name:
+password:
+endpoint: the ip address and the port number
+            which hil is running on
+status_file: it should be /var/lib/leasing
+
+******************
+
+Status file format (var/lib/leasing):
+node_name project_name current_time
+
+current time is the number of times that cron jon
+has been called.
+"""
+
+# from sets import Set
+import time
+import ConfigParser
+from os import remove
+from shutil import move
+
+from hil.client.client import Client, RequestsHTTPClient
+from hil.client.base import FailedAPICallException
+
+
+class HILClientFailure(Exception):
+    pass
+
+
+class StatusFileError(Exception):
+    pass
+
+
+def hil_client_connect(endpoint_ip, name, pw):
+
+    hil_http_client = RequestsHTTPClient()
+    """
```

Comment:
Is this commented out for debug purposes?  It would be helpful to know if the client wasn't able to connect.

Reply:
I just saw Ian's older comment, if this won't be a likely failure case we can probably just remove it entirely.

Reply:
I will remove it, just wanted to be sure that other reviewers have the same opinion.